src/InverseSolve/models.jl

```diff
@@ -25,6 +25,11 @@ function fMod(m;f::Function=identity,df::Function=m->ones(length(m)))
 	return f(m),sdiag(df(m))
 end
 
+function dummyMod(m)
```

Comment:
fMod creates a sparse identity. This is done for each worker in pFor. This identity corresponds to the large mesh (i think). If we have many pFor's, this may create a computational bottleneck. Sparse matrices are expensive (memory, serial multiplication etc.). Someone mentioned an identity matrix in Julia (I think Eye?) that is implemented using a scalar behind the scene, but I couldn't find it. dummyMod uses a scalar which is much less harmful (most of our usage is single-physics).